build.gradle.kts

```diff
@@ -7,6 +7,7 @@ plugins {
 
 buildscript {
```

Comment:
Use `plugins { ... }` block. If you need to declare custom repositories for plugins, you can use `pluginManagement` block in `settings.gradle.kts`

Reply:
Actually, let me handle this after switching over to `Gradle Version Catalog` (also created a card for this on the dashboard)